code/style.css

```diff
@@ -1,3 +1,280 @@
 body {
-  background: #FFECE9;
+  background-color: #f0f0f0;
+  margin: 0;
+  padding: 0;
+  font-family: 'Barlow', sans-serif;
+}
+
+/*header*/
+.header {
+  background-color: #7aa094;
+  display: grid;
+  grid-column: span 12;
+  justify-content: center;
+}
+
+.header-text {
+  color: #fff;
+  display: grid;
+  grid-column: span 2/8;
+  font-weight: 300;
+  font-size: 25px;
+}
+
+form#search-form {
+  grid-column: span 12;
+  margin-bottom: 10px;
+}
+
+.icon_header {
+  padding: 10px 0px 10px 0px;
+  height: 50px;
+  display: inline;
+  display: grid;
+  grid-column: span 8/12;
+}
+
+/*search field*/
+.search-container {
+  grid-template-columns: repeat(12, 1fr);
+  grid-column: span 12;
+}
+
+.input-field {
+  margin: 7px 0px 8px 0px;
+  grid-column: 1/12;
+  padding: 10px 20px;
+  border-radius: 15px;
+  font-size: 15px;
+  color: #7aa094;
+  border: 1px solid #7aa094;
+  font-family: 'Barlow', sans-serif;
+  font-weight: 500;
+}
+
+.button-search {
+  margin: 7px 0px 7px 0px;
+  grid-column: 2/4;
+  padding: 10px 20px;
+  border-radius: 15px;
+  color: #fff;
+  background-color: #56786d;
+  font-size: 15px;
+  border: 1px solid #7aa094;
+  font-family: 'Barlow', sans-serif;
+  font-weight: 500;
+}
+
+.button-search:hover {
+  color: #364b45;
+  background-color: #56786d;
+  font-weight: 500;
+}
+
+/*main grid*/
+.main-grid {
+  display: grid;
+  grid-template-columns: repeat(12, 1fr);
+}
+
+.grid-container {
+  display: grid;
+  grid-template-columns: repeat(12, 1fr);
+  grid-column: span 12;
+  column-gap: 10px;
+  row-gap: 10px;
+}
+
+/*profile*/
+.my-profile {
+  background-color: #f0f0f0;
+  display: grid;
+  grid-template-columns: repeat(12, 1fr);
+  grid-column: span 12;
+  border-radius: 25px 0px;
+  margin: 15px 0px 15px 0px;
+}
+
+.profileinfo {
+  background-color: #f0f0f0;
+  grid-column: span 2/4;
+}
+
+.profileinfo a {
+  color: black;
+  text-decoration: none;
+}
+
+.profileimg {
+  grid-column: 6/12;
+}
+
+.profileimg img {
+  width: 100%;
+  height: 100%;
+  object-fit: cover;
+  border-radius: 25px 0px;
+}
+
+/*repos*/
+.box-repo {
+  padding: 10px;
+  background-color: #7aa094;
+  display: grid;
+  grid-column: span 12;
+  border-radius: 25px 0px;
```

Comment:
A small detail that makes the boxes look really good in the mobile version I think :)